BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/BTConnector_Discovery.java

```diff
@@ -16,16 +16,12 @@
  */
 public class BTConnector_Discovery implements WifiServiceSearcher.DiscoveryInternalCallBack {
 
-    BTConnector_Discovery that = this;
-
-    static String JSON_ID_PEERID   = "pi";
-    static String JSON_ID_PEERNAME = "pn";
-    static String JSON_ID_BTADRRES = "ra";
+    private final BTConnector_Discovery that = this;
 
     public interface  DiscoveryCallback{
```

Comment:
Basically wanted the plug-in to be called only from the BtConnector class instance, thus other parts are reporting to it.